### DISCUSSION TITLE
Suggestion: Identify Commercial Formule and Casks  With an Indicator Flag

### DISCUSSION THREAD

**Discussion by mvsopen** [Homebrew/discussions#1614] [Tap maintenance and Homebrew development]
Brew is a fantastic resource, and I use it, daily. However, I'm seeing an increasing number for formule and casks which have some type of fee required after the demo period expires.

I've got nothing against commercial software developers, but can these programs be somehow flagged when you list their descriptions in brew.sh? Example "xxx - A great commercial program    $"   (The "$" symbol would make a good flag for those apps and tools which require a fee.

**Comment by SMillerDev** [comment]
For formulae we're trying to document their licenses and removing them if they're not OSI compatible so if you found any that are paid please make a pull request to remove them.

As for casks, I don't think it's feasible to keep track of their licensing model. We won't have any way to automate this so it would all be on maintainers and contributors to look up.

**Reply by tjluoma** [reply]
Maybe the idea that software developers should give their work away for free should die off, and people should expect to pay for software just like they pay for anything else.

If that was the default expectation, then when you encounter a free app, you can simply enjoy it as a delightful exception.

**Reply by SMillerDev** [reply]
For commercial software, absolutely. But I wish Open Source a long and happy journey onward :smile:

**Reply by tjluoma** [reply]
Same here… my only point was that we should stop _assuming_ that free is the default.

Oh, and just to put my money where my (big) mouth is, I signed up as a Patreon for Homebrew, which I should have done a long time ago.

**Reply by cho-m** [reply]
On Cask-side, I think this falls into https://docs.brew.sh/Acceptable-Casks#homebrew-cask-is-not-a-discoverability-service
> we’re not a discoverability service and our users are expected to have reasonable knowledge about the apps they’re installing through us before doing so

In terms of license info, the `license` stanza was removed from Casks as detailed in: Homebrew/homebrew-cask#16692
